util/gh-pages/script.js

```diff
@@ -536,12 +321,34 @@ function addListeners() {
         if (event.target.classList.contains("copy-to-clipboard")) {
             copyToClipboard(event);
         } else if (event.target.classList.contains("anchor")) {
-            lintAnchor(event);
+            event.target.closest("article")
+                .querySelector(`input[type="checkbox"]`)
+                .checked = true;
         }
     });
 
+    document.getElementById("filter-clear").addEventListener("click", () => {
+        elements.search.value = "";
+        searchState.filterLints();
+    })
+
+    addResetListeners(".reset-all", true);
+    addResetListeners(".reset-none", false);
+    addResetListeners(".reset-default", null);
+
+    document.getElementById("reset-versions").addEventListener("click", () => {
+        for (const input of Object.values(elements.versions)) {
+            input.value = "";
+        }
+        filters.filterLints();
+    });
+
     document.addEventListener("keypress", handleShortcut);
     document.addEventListener("keydown", handleShortcut);
+
+    document.querySelectorAll(".dropdown").forEach(setupDropdown);
```

Comment:
Please use `onEachLazy` instead of `forEach`. Chrome recomputes the list at every change made to the DOM.